wiki/Glossary/en.md

```diff
@@ -8,6 +8,10 @@ Throughout the years, the osu!community has adopted its own set of slang that ma
 
 If the name is not obvious enough, the abandoned wasteland is a hidden sub-forum where the instructions to the moderators concerning it will: "Put stuff we do not really need in here." When a thread is or is said to be "wastelanded", it is moved there.
 
+### Accuracy
+
+A [game metric](/wiki/Accuracy) to measure the player's rhythmical accuracy. Sometimes shortened to _Acc_.
+
 ### AFK
```

Comment:
basic internet acronym, doesn't really fit with the rest of the osu specific terms, should remove

Reply:
Maybe there should be a category for "internet acronyms" used in the game? I think it's fair to have them